src/ch01-01-installing-tools.md

```diff
@@ -3,13 +3,34 @@
 ### What is Clarinet?
 Clarinet is a Clarity runtime packaged as a command line tool, designed to facilitate smart contract understanding, development, testing and deployment. Clarinet consists of a Clarity REPL and a testing harness, which, when used together allow you to rapidly develop and test a Clarity smart contract, with the need to deploy the contract to a local devnet or testnet.
 
-Clarity is a decidable smart contract language that optimizes for predictability and security, designed for the Stacks blockchain. Smart contracts allow developers to encode essential business logic on a blockchain.
+Clarity is a decidable smart contract language that optimises for predictability and security, designed for the Stacks blockchain. Smart contracts allow developers to encode essential business logic on a blockchain.
 
 ### Install on macOS (Homebrew)
+This process relies on the macOS package manager called [Homebrew](https://brew.sh/). Using the `brew` command you can easily add powerful functionality to your mac, but first we have to install it.
 
+To get started, launch your [Terminal](https://support.apple.com/en-ph/guide/terminal/welcome/mac) (/Applications/Utilities/Terminal) application. Terminal is a versatile command line system that comes with every Mac computer.
+
+If you don't already have [XCode](https://developer.apple.com/xcode/) installed, it's best to first install the command line tools as these will be used by homebrew:
```

Comment:
>don't

Nit: I do not think we use any contractions in the book. Correct me if I am wrong though.

Reply:
It is true. Updated the word.